README.rst

```diff
@@ -1,196 +0,0 @@
-`Documentation`_ | Blog_ |  `View on Github`_ |  `Download Stable ZIP`_  |  `Download Stable TAR`_
```

Comment:
Pls do not remove this file. If possible stash the add back commit into the one that removed it.

Reply:
Hm, file restored. I already squashed. I will squash at the and too, right? Then an extra commit now does not matter.